cumulusci/cumulusci.yml

```diff
@@ -2,6 +2,11 @@ cumulusci:
     keychain: cumulusci.core.keychain.EncryptedFileProjectKeychain
 
 tasks:
+    assign_permission_set:
+        description: Assign Permission Set specified in the "extra" option using [--permsetname "Permission_Set_DeveloperName"]
+        class_path: cumulusci.tasks.sfdx.SFDXOrgTask
+        options:
+            command: 'force:user:permset:assign'
```

Comment:
Since --permsetname is always required (I think), can you change the command option's value to 'force:user:permset:assign --permsetname ' so the only thing you have to do in the extra option is specify the permset's name?  That would make the cci command much nicer: `cci task run assign_permission_set -o extra Permission_Set_DeveloperName` and I think would help avoid issues with cci trying to parse the --permsetname as an option to the task run command.

Reply:
Thanks, @jlantz ! I've changed the command now - also added the 'Salesforce' group so that it shows up nicely in the task list command.